gpAux/gpperfmon/src/gpmon/gpperfmon_install

```diff
@@ -164,35 +165,63 @@ if __name__ == '__main__':
         cmd = Command("""PGPORT=%d psql template1 -c "DROP ROLE IF EXISTS gpmon" """ % options.port)
         commands.append(cmd)
 
-        cmd = Command("""PGPORT=%d psql template1 -c "CREATE ROLE gpmon WITH SUPERUSER CREATEDB LOGIN ENCRYPTED PASSWORD '%s'" """ % (options.port, options.password))
+        if options.password:
+            cmd = Command("""PGPORT=%d psql template1 -c "CREATE ROLE gpmon WITH SUPERUSER CREATEDB LOGIN ENCRYPTED PASSWORD '%s'" """ % (options.port, options.password))
+        else:
+            cmd = Command("""PGPORT=%d psql template1 -c "CREATE ROLE gpmon WITH SUPERUSER CREATEDB LOGIN" """ % options.port)
         commands.append(cmd)
 
-        cmd = Command("""echo "local    gpperfmon         gpmon         md5" >> %s""" % pg_hba, showOutput=True)
-        commands.append(cmd)
+        if options.password:
```

Comment:
may be combine this if with previous one?

Reply:
I agree that this could make code smaller, but now it's organized logicaly, each if states it's own logical task. 
It could have deeper refactoring but it's out of the scope of this task.